jwst/extract_1d/extract.py

```diff
@@ -458,10 +458,16 @@ def find_dispaxis(input_model, slit, spectral_order, extract_params):
             wl_10 = stuff[2]
         dwlx = wl_10 - wl_00
         dwly = wl_01 - wl_00
-        if (np.isnan(dwlx) or np.isnan(dwly) or
+        if (np.isnan(dwlx) or np.isnan(dwly) or dwlx == dwly or
             wl_00 == 0 or wl_01 == 0 or wl_10 == 0):
-                log.warning("wavelength from WCS is NaN or 0 "
-                            "within the bounding box")
+                if dwlx == dwly:
```

Comment:
Does this also trigger a full bounding box computation like the nan/zero case?

Reply:
Yes.  This is already included above, in line 461.  I added this separate test just so I could log a different message from the nan/zero case.

Reply:
got it.